wear/src/main/java/io/homeassistant/companion/android/settings/SettingsComposable.kt

```diff
@@ -166,3 +217,160 @@ fun ScreenSetFavorites(
         }
     }
 }
+
+@Composable
+fun ScreenSetTileShortcuts(
+    shortcutEntities: MutableList<String>,
+    context: Context,
+    swipeDismissableNavController: NavHostController,
+    onShortcutEntitySelectionChange: (Int) -> Unit
+) {
+
+    val scalingLazyListState: ScalingLazyListState = rememberScalingLazyListState()

```

Comment:
If you add `RotaryEventState(scrollState = scalingLazyListState)` in between 229 and 230 then the bezel will work with rotary input :)

Same thing if you apply it to the other lazy list :)

Reply:
Awesome, works like a charm!